pygmt/helpers/decorators.py

```diff
@@ -180,8 +180,9 @@ def use_alias(**aliases):
     R = bla J = meh
     >>> my_module(region='bla', projection='meh')
     R = bla J = meh
-    >>> my_module(region='bla', projection='meh', J="bla")
-    # doctest: +NORMALIZE_WHITESPACE
+    >>> my_module(
+    ...    region='bla', projection='meh', J="bla"
+    ... )  # doctest: +NORMALIZE_WHITESPACE
```

Comment:
Thanks for the change. My old commit worked for me locally, but failed in the CI. Perhaps I'm using a different pylint verison.